lib/jsonapi_compliable.rb

```diff
@@ -109,6 +109,7 @@ def data
 if defined?(Rails)
   require 'jsonapi_compliable/railtie'
   require 'jsonapi_compliable/rails'
+  require 'jsonapi_compliable/responders'
```

Comment:
Not everyone uses the responders gem. I suggest we do a separate conditional check for the presence of the gem.  Can do the same in the generator so that we generate either a simple `respond_with` or a more verbose `respond_to do |format|` block depending on presence.

Reply:
This is just loading a 19 line file, which will not break even if you don't have that gem. I'm not sure it's worth the extra hop of adding to a railtie or forcing a manual require.

Agree on the generator though, good point